### ISSUE TITLE
Can't install due to missing dependency package

### ISSUE THREAD

**Issue by gavinltomra** [OSInside/kiwi-boxed-plugin#73]
From OpenSUSE Leap 15.5 (with the Virtualization:Appliances:Builder repo added), I'm trying to run the following:

```
sudo zypper in python3-kiwi_boxed_plugin
```

But am met with:

```
Problem: 1: nothing provides 'python3-progressbar2' needed by the to be installed python3-kiwi_boxed_plugin-0.2.37-lp155.1.1.noarch
 Solution 1: do not install python3-kiwi_boxed_plugin-0.2.37-lp155.1.1.noarch
 Solution 2: break python3-kiwi_boxed_plugin-0.2.37-lp155.1.1.noarch by ignoring some of its dependencies
```

Oddly there does appear to be a [matching package](https://build.opensuse.org/package/show/Virtualization:Appliances:Builder/python-progressbar2) in the build system but it doesn't exist in the download repository.  Is another repository required now too?  I didn't find any info about this in the docs.

**Comment by gavinltomra** [comment]
So I instead tried:

```
sudo zypper in python3-pip
sudo pip3 install kiwi-boxed-plugin
```

And this installed, but on first attempt to actually run it, it just says:

```
[ INFO    ]: 03:21:01 | Reading box plugin config file: /usr/lib/python3.6/site-packages/kiwi_boxed_plugin/config/kiwi_boxed_plugin.yml
[ ERROR   ]: 03:21:01 | KiwiBoxPluginConfigError: Failed to validate /usr/lib/python3.6/site-packages/kiwi_boxed_plugin/config/kiwi_boxed_plugin.yml: {'box': [{4: [{'arch': [{3: [{'name': ['unallowed value ppc64le']}]}]}]}]}
```
Not sure why it's even looking at `ppc64le`; I was trying to use `--box leap` as per the docs.

Side note: following the pip install, `kiwi-ng help` reports that no man page was installed.

**Comment by hrfried** [comment]
I'm also seeing this running from tumbleweed, using the version from Virtualization:Appliances:Builder repo

**Comment by hrfried** [comment]
@gavinltomra Just a heads up, reverting back to v0.2.36 worked fine for me, if you're in a pinch and want a quick fix. I just rebuilt with osc locally and installed the resulting RPM, but you could do it a number of different ways.

**Comment by gavinltomra** [comment]
For the moment I've ended up using the version of Kiwi in the Leap 15.3 repository (without using the boxed plugin for now), because I kept getting weird package version conflicts between these and the Builder repo, despite me setting the Builder repo to a higher priority.  It's working well enough, though it complains about BDB/NDB mismatches.

**Comment by schaefi** [comment]
>KiwiBoxPluginConfigError: Failed to validate /usr/lib/python3.6/site-packages/kiwi_boxed_plugin/config/kiwi_boxed_plugin.yml

yes that was my fault, sorry for this one. It got fixed with commit #61dc5aa7

So you should not see this in v0.2.38 anymore

**Comment by schaefi** [comment]
regarding the package dependency to the progressbar I build that package in Virtualization:Appliances:Builder but only for the following targets 

```
openSUSE_Leap_15.5   x86_64     succeeded
openSUSE_Leap_15.5   aarch64    succeeded
openSUSE_Leap_15.5   ppc64le    succeeded
openSUSE_Leap_15.5   s390x      succeeded
ALP                  x86_64     succeeded
```

So you are saying we need it also for other distros ?

**Comment by schaefi** [comment]
I guess this is solved

**Comment by gavinltomra** [comment]
It's come back now (Leap 15.5 appliance builder repo):

```
Problem: 1: nothing provides 'python311-progressbar2' needed by the to be installed python311-kiwi_boxed_plugin-0.2.38-lp155.2.1.noarch
 Solution 1: do not install python311-kiwi_boxed_plugin-0.2.38-lp155.2.1.noarch
 Solution 2: break python311-kiwi_boxed_plugin-0.2.38-lp155.2.1.noarch by ignoring some of its dependencies
```

**Comment by schaefi** [comment]
Thanks, yeah the py3.11 stack in Leap had some side effects. I'm going to fix the packaging

**Comment by schaefi** [comment]
[   22s] RPMS/noarch/python311-progressbar2-4.0.0-lp155.11.1.noarch.rpm

should be found now
